include/openssl/evp.h

```diff
@@ -269,7 +269,7 @@ int (*EVP_CIPHER_meth_get_ctrl(const EVP_CIPHER *cipher))(EVP_CIPHER_CTX *,
 # define         EVP_CIPH_CUSTOM_COPY            0x400
 /* Don't use standard iv length function */
 # define         EVP_CIPH_CUSTOM_IV_LENGTH       0x800
-/* Allow use default ASN1 get/set iv */
+/* Legacy and no longer relevant: Allow use default ASN1 get/set iv */
 # define         EVP_CIPH_FLAG_DEFAULT_ASN1      0x1000
```

Comment:
Shouldn't we just make this 0?

Reply:
I was pondering that.  I was also pondering giving `EVP_CIPH_FLAG_CUSTOM_ASN1` that value instead, but decided against it, seeing possible confusion...

Reply:
Fixed